vocabularies/Analytics.json

```diff
@@ -211,6 +211,13 @@
             "$Type": "Analytics.AutoExpandEntry",
             "@Core.Description": "Entries with exceptional expansion",
             "@Core.OptionalParameter": {}
+          },
+          {
+            "$Name": "SubtotalsAtBottom",
+            "$Type": "Edm.Bool",
```

Comment:
@simlin Are two value sufficient? Or could we have three (on top, at bottom, both)?